tools/osbuild-mpp

```diff
@@ -811,6 +880,9 @@ class ManifestFileV2(ManifestFile):
         files = element_enter(self.sources, "org.osbuild.curl", {})
         self.source_urls = element_enter(files, "items", {})
 
+        inline_files = element_enter(self.sources, "org.osbuild.inline", {})
```

Comment:
This is in the `ManifestFileV2` object, I need to check if it works on the V1 (and thus can be moved to the parent object) or if it needs to be adjusted